salt/redash/init.sls

```diff
@@ -1,16 +1,24 @@
+{% from 'lib.sls' import create_pg_database %}
+
 include:
   - docker
 
 # =================================================================================================
 # See https://ocdsdeploy.readthedocs.io/en/latest/deploy/redash.html for installation instructions.
 # =================================================================================================
 
-redash requirements:
-  pkg.installed:
-    - pkgs:
-      - pwgen
-      - postgresql-client-10
-
-/opt/redash:
-  file.directory:
+# Create Redash configuration and directories
```

Comment:
Is it possible/appropriate to user `docker_apps` like in `registry/init.sls` ? If so, then the env file would instead be configured mostly in Pillar directly.

I haven't yet written up the docs page for it. #309

Reply:
I have updated redash to use `docker_apps`.